source/game_sa/CarEnterExit.cpp

```diff
@@ -642,18 +639,10 @@ void CCarEnterExit::QuitEnteringCar(CPed* ped, CVehicle* vehicle, int32 doorId,
             vehicle->SetGettingInFlags(8);
             break;
         case 10:
-            if (vehicle->m_nMaxPassengers) {
-                vehicle->SetGettingInFlags(1);
-            } else {
-                vehicle->SetGettingInFlags(3);
-            }
+            vehicle->m_nMaxPassengers ? vehicle->SetGettingInFlags(1) : vehicle->SetGettingInFlags(3);
             break;
         case 11:
-            if (vehicle->m_nMaxPassengers) {
-                vehicle->SetGettingInFlags(2);
-            } else {
-                vehicle->SetGettingInFlags(3);
-            }
+            vehicle->m_nMaxPassengers ? vehicle->SetGettingInFlags(2) : vehicle->SetGettingInFlags(3);
```

Comment:
Thats not exactly what I meant....
```suggestion
            vehicle->SetGettingInFlags(vehicle->m_nMaxPassengers ? 2 : 3);
```

Reply:
fix this, and that should be about it